ts/features/bonus/siciliaVola/utils.ts

```diff
@@ -0,0 +1,41 @@
+import I18n from "../../../i18n";
+import { SvVoucher } from "./types/SvVoucher";
+
+export const fromVoucherToDestinationLabels = (voucher: SvVoucher) => {
+  switch (voucher.category) {
```

Comment:
```suggestion
  switch (voucher.category) {
  case "disabled":
      return [];
```
To ensure the compiler checks if you handle all possible cases, I suggest to remove the final return and put also the "disabled" case. In this way if you add a category and you won't update this switch, the complier will raise an error

https://www.fullstory.com/blog/discriminated-unions-and-exhaustiveness-checking-in-typescript/

Reply:
Thank you for the suggestion. Modified in [066bf0a](https://github.com/pagopa/io-app/pull/3400/commits/066bf0ad6ad3edf59cbde816ebe5ce2da757ff79)